gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
I guess what I am trying to say is those two are two separate components, and `require_relative` couples them. I am still not convinced :)